### PR TITLE
Keys: add primary address to seed & keys page

### LINKED ISSUE #2434: No Primary address in "Seed and Keys" page
Normally when you want to know the seed and keys, you also want to know the main address, since you need the main address + private view-key + private spend-key in order to restore a wallet.

Therefore it would be nice to have all important information about the wallet in a single page.

### PR DESCRIPTION
**Pull request by rating89us**
Closes #2434

image

### CODE REVIEW

**Comment by xiphon** [comment]
Maybe place it right before the Keys section?

File: pages/Keys.qml [hunk 1]
```diff
@@ -140,6 +140,15 @@ Rectangle {
...
+                Layout.fillWidth: true
+                id: primaryAddress
+                readOnly: true
+                copyButton: true
+                wrapMode: Text.Wrap
+                labelText: qsTr("Primary address") + translationManager.emptyString
+                fontSize: 16
+            }           
             MoneroComponents.LineEdit {
                 Layout.fillWidth: true
```
Comment (selsta) [inline pages/Keys.qml]:
```suggestion
                Layout.fillWidth: true
                Layout.topMargin: 25
```

File: pages/Keys.qml [hunk 2]
```diff
@@ -300,4 +311,3 @@ Rectangle {
 
 
 
-
```
Comment (selsta) [inline pages/Keys.qml]: Please try to avoid unrelated whitespace changes, looks good otherwise.
